src/index.js

```diff
@@ -26,15 +26,44 @@ class ServerlessLeo {
       create: {
         commands: {
           bot: {
-            usage: 'Create a nodejs microservice',
+            usage: 'Create a leo bot',
             lifecycleEvents: [
               'copy-template',
               'replace-tokens'
             ],
             options: {
               name: {
-                usage: 'Specify the name of the bot',
-                shortcut: 'n'
+                usage: 'Name of the bot',
+                type: 'string',
+                shortcut: 'n',
+                required: true
+              },
+              language: {
+                usage: 'Programming language of the bot. Defaults to node',
+                type: 'string',
+                shortcut: 'l',
+                default: 'node'
+              },
+              path: {
+                usage: `Output path for the bot. Defaults to bots${path.sep}{name}`,
+                type: 'string',
+                shortcut: 'p'
+              },
+              type: {
+                usage: 'Stream type of the bot. Defaults to load',
+                type: 'string',
+                shortcut: 't',
+                default: 'load'
+              },
+              'read-queue': {
+                usage: 'Read queue. Defaults to {name}_read',
+                type: 'string',
+                shortcut: 'q' // -r reserved for aws-region
+              },
+              'write-queue': {
```

Comment:
in the template this is called destination so I think we should keep it consistent with either `destination`, `destination_queue`